client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -16,7 +16,8 @@ import {
 	getImageEditorTransform,
 	getImageEditorFileInfo,
 	getImageEditorCrop,
-	isImageEditorImageLoaded
+	isImageEditorImageLoaded,
+	getImageMeetsMinimumDimensions
```

Comment:
We tend to prefix boolean getters with, `has` or `is`, or `can`. If original w/h dimensions are not available, do you think that would be more useful to store? This value seems like something we can derive in the component, and wouldn't be used elsewhere.